build.rs

```diff
@@ -0,0 +1,14 @@
+fn main() {
```

Comment:
Please format the code in this file as well.

Reply:
I believe it is formatted.

Reply:
It's using tabs instead of spaces.